### PR TITLE
add cc_resource_type_detail field in all tweets

### PR DESCRIPTION
**Pull request by apoorvsingal**
Schema of the field:
```
{
  resource_types: array<"Bed" | "ICU" | "Ventilator" | "Oxygen Bed" | "Remdesivir" | "Favipiravir" | "Tocilizumab" | "Plasma" | "Food" | "Ambulance" | "Oxygen Cylinder" | "Oxygen Concentrator" | "Covid Test" | "Helpline">,
  categories: array<"hospital" | "ambulance" | "medical device" | "test" | "helpline">,
  verification_status: "Verified" | "Not Verified" | "Unknown",
  verified_at: string (time)
  locations: array<{ city: string, state: string }>,
  phone_numbers: array<string>,
  emails: array<string>
}
```
Example: 

image

Note: I didn't test it for the v2 API.

### CODE REVIEW

File: pages/api/data.js [hunk 1]
```diff
@@ -12,6 +13,18 @@ export default async (req, res) => {
       max_results: max_results || 100
     });
 
+    if(apiResponse.res.data){
+      const data = [];
+    
+      for(let status of apiResponse.res.data){
+        statuses.push({
```
Comment (brainless) [inline pages/api/data.js:20]: Where is `statuses` defined?

**Review by brainless — CHANGES_REQUESTED** [review]
@apoorvsingal `statuses` is not defined in `pages/api/2/data.js` either. Also, I added them but I get zero results for v2 (with or without the version in URL). v1.1 seems to be working fine but I have not checked the actual `cc_resource_type_detail` field yet.

Thanks

**Comment by apoorvsingal** [comment]
Sorry, my bad. I'll fix.

File: pages/api/data.js (hunk 1 above)
Comment (apoorvsingal) [reply pages/api/data.js]: It's supposed to be `data` instead of `statuses`. My bad.

**Comment by tomahawk-pilot** [comment]
#### 📝Changed routes:



- [**/api/1.1/data**](https://covidconnect-g3fi1jki7-coronasafe.vercel.app/api/1.1/data)
- [**/api/2/data**](https://covidconnect-g3fi1jki7-coronasafe.vercel.app/api/2/data)
- [**/api/data**](https://covidconnect-g3fi1jki7-coronasafe.vercel.app/api/data)



Commit 9a46df4b43d924891ce92ba731d3de9871d03741 (https://covidconnect-g3fi1jki7-coronasafe.vercel.app).

**Comment by viksit** [comment]
any updates?

**Comment by brainless** [comment]
Looks good to me.

**Pull request merged**